test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNet/BulkOperation/BulkOperationTest.cs

```diff
@@ -100,12 +100,14 @@ public async Task PatchEmployee_WithDeletedAndODataId_IsSerializedSuccessfully()
             }
         }
 
-        [Fact]
-        public async Task PostEmployee_ExpandNavigationProperties()
+        [Theory]
+        [InlineData("/convention/Employees")]
+        [InlineData("/convention/Employees?$expand=Friends,NewFriends($expand=NewOrders)")]
```

Comment:
The first scenario we don't have the query option set explicitly but in the second scenario we do. So in the first case, they just need to set EnableQuery on their post action method,, and we'll generate the expand string and apply it as expected.

Reply:
Is there a test for when the `EnableQuery` is not set from the controller action?

Reply:
We don't need a test for when `EnableQuery` is not set because the response will not have navigation properties expanded.

Reply:
The code changes to enable this are in the EnableQuery action filter